src/common/options/mon.yaml.in

```diff
@@ -764,6 +764,16 @@ options:
   services:
   - mon
   with_legacy: true
+- name: mds_beacon_mon_down_grace
+  type: float
+  level: advanced
+  desc: tolerance in seconds for missed MDS beacons to monitors
+  fmt_desc: The interval without beacons before Ceph declares an MDS laggy
+    when a monitor is down.
+  default: 60
+  services:
+  - mon
```

Comment:
this part is not necessary. see https://docs.ceph.com/en/latest/dev/config/#service